boards/arm/nrf51_pca10031/doc/nrf51_pca10031.rst

```diff
@@ -0,0 +1,158 @@
++===========+============+======================+
++-----------+------------+----------------------+
+| ADC       | on-chip    | adc                  |
++-----------+------------+----------------------+
+| CLOCK     | on-chip    | clock_control        |
++-----------+------------+----------------------+
+| FLASH     | on-chip    | flash                |
++-----------+------------+----------------------+
+| GPIO      | on-chip    | gpio                 |
++-----------+------------+----------------------+
+| I2C(M)    | on-chip    | i2c                  |
++-----------+------------+----------------------+
+| NVIC      | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| PWM       | on-chip    | pwm                  |
++-----------+------------+----------------------+
+| RADIO     | on-chip    | Bluetooth            |
++-----------+------------+----------------------+
+| RTC       | on-chip    | system clock         |
++-----------+------------+----------------------+
+| RTT       | Segger     | console              |
++-----------+------------+----------------------+
+| SPI(M/S)  | on-chip    | spi                  |
++-----------+------------+----------------------+
+| UART      | on-chip    | serial               |
++-----------+------------+----------------------+
+| WDT       | on-chip    | watchdog             |
++-----------+------------+----------------------+
+
+Other hardware features are not supported by the Zephyr kernel.
+See `nRF51 Dongle website`_ and `Nordic Semiconductor Infocenter`_
+for a complete list of nRF51 Dongle hardware features.
+
+Connections and IOs
+===================
+
+LED
+---
+
+* LED1 (red)   = P0.21
+* LED1 (green) = P0.22
+* LED1 (blue)  = P0.23
+
+Push buttons
+------------
+
+* BOOT = SW1 = boot/reset
+
+Programming and Debugging
+*************************
+
+Flashing
+========
+
+Follow the instructions in the :ref:`nordic_segger` page to install
+and configure all the necessary software. Further information can be
+found in :ref:`nordic_segger_flashing`. Then build and flash
+applications as usual (see :ref:`build_an_application` and
+:ref:`application_run` for more details).
+
+Here is an example for the :ref:`hello_world` application.
+
+First, run your favorite terminal program to listen for output.
+
+.. code-block:: console
+
+   $ minicom -D <tty_device> -b 115200
+
+Replace :code:`<tty_device>` with the port where the board nRF51 Dongle
+can be found. For example, under Linux, :code:`/dev/ttyACM0`.
+
+Then build and flash the application in the usual way.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: nrf51_pca10031
+   :goals: build flash
+
+Debugging
+=========
+
+Refer to the :ref:`nordic_segger` page to learn about debugging Nordic boards with a
+Segger IC.
+
+
+Testing the LEDs on the nRF51 Dongle
+************************************
+
+There are 1 sample that allow you to test that the LED on
```

Comment:
Let's change this whole section into:
```
Build and flash the :ref:`blinky-sample` sample to test that the onboard LED
is working properly with Zephyr.  LED definitions can be found in
:file:`boards/arm/nrf51_pca10031/board.h`.
```

Reply:
Actual this ```board.h``` does not exist anymore so I will remove this reference.